server/app/models/user.rb

```diff
@@ -12,11 +12,10 @@ def email_activate
   end
 
   private
-  def confirmation_token
+  def set_confirm_token
```

Comment:
@ryanzesch, I changed the name since I found it a bit confusing during our heroku bug hunt. LMK if you want it reverted.

Reply:
All good by me :)